docs/Operators.md

```diff
@@ -9408,6 +9409,91 @@ expect(
 </details>
 
 
+### <a name="Gelu"></a><a name="gelu">**Gelu**</a>
+
+  Gelu takes one input data (Tensor<T>) and produces one
+  output data (Tensor<T>) where the gaussian error linear units function,
+  `y = 0.5 * x * (1 + erf(x/sqrt(2)))` is applied to the tensor elementwise.
+  If the attribute "approximate" is set to "tanh", the function estimation,
+  `y = 0.5 * x * (1 + Tanh(sqrt(2/π) * (x + 0.044715 * x^3)))` is used and applied
+  to the tensor elementwise.
+
+
+#### Version
+
+This version of the operator has been available since version 20 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>approximate</tt> : string (default is none)</dt>
+<dd>Gelu approximation algorithm: tanh, none(default).'none': do not use approximation.'tanh': use tanh approximation.</dd>
```

Comment:
```suggestion
<dd>Gelu approximation algorithm: `"tanh"`, `"none"` (default). `"none"`: do not use approximation. `"tanh"`: use tanh approximation.</dd>
```

Reply:
Made these changes in onnx/defs/math/defs.cc and recompiled to update Operators.md